### PR TITLE
UX-620 My name is Peppa

### PR DESCRIPTION
**Pull request by nickine**
#### What does this PR do? (please provide any background)
Create the Peppa Pig Brand

#### How can this be tested?
`cd pattern-library && npm link ../brands/peppa`
`cd pattern-library && npm link ../brands/paultons` (because of mapping association)
`cd pattern-library && npm link ../brands/globals`

#### Any tech debt?
Shouldn't be

#### Screenshots / Screencast
screencapture-localhost-8687-playground-shortbreaks-availability-index-html-1484662748323
screencapture-localhost-8687-playground-shortbreaks-payment-old-index-html-1484662806058
screencapture-localhost-8687-playground-shortbreaks-upgrades-index-html-1484662759419

#### What gif best describes how you feel about this work?
gif

- [x] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Reviewer**
- [ ] :+1:
- [ ] UXUI Team :+1:
- [ ] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked all the work locally on pattern library.

---

### CODE REVIEW

File: brands/paultons/src/less/footer.less [hunk 1]
```diff
@@ -12,17 +12,17 @@
...
   margin-bottom: -20px;
   position: relative;
 
   .tile-icons {
     display: inline-block;
     background-color: @color-white;
-    color: rgb(33, 33, 33);
-    padding: 10px;
-    border-radius: 3px;
+    color: initial;
```
Comment (jodiedoubleday) [inline brands/paultons/src/less/footer.less]: initial? not seen this used int he wild before, be warned it only works for later than IE11

File: brands/peppa/src/less/backgrounds.less [hunk 2]
```diff
@@ -1 +1,26 @@
...
+
+//
+// Contextual backgrounds
+// --------------------------------------------------
+
+.bg-normal {
+  .background-variant(@bg: @bg-default);
+}
+.bg-default {
+  background-color: @bg-default;
```
Comment (jodiedoubleday) [inline brands/peppa/src/less/backgrounds.less]: could you not use the same mixin as `bg-normal` above? (I mean its not necessary but to keep consistency.)

File: brands/peppa/src/less/backgrounds.less [hunk 3]
```diff
@@ -1 +1,26 @@
...
+}
+.bg-brand {
+  background-color: @bg-brand;
```
Comment (jodiedoubleday) [inline brands/peppa/src/less/backgrounds.less]: as above?

File: brands/peppa/src/less/colours.less [hunk 4]
```diff
@@ -1 +1,63 @@
 @import '../../../brand-globals/src/less/colours';
+//
+// Colors
+// --------------------------------------------------
+
+// Core brand colors
+@logo-primary:             black;
```
Comment (jodiedoubleday) [inline brands/peppa/src/less/colours.less]: Im surprised the logo primary isn't green or pink

File: brands/peppa/src/less/footer.less [hunk 5]
```diff
@@ -1 +1,30 @@
 @import '../../../brand-globals/src/less/footer';
+
+.secure-footer {
```
Comment (jodiedoubleday) [inline brands/peppa/src/less/footer.less]: in the future we might look to making the HX footer markup similar to shortbreaks or visa versa. but good idea to put brands specific one for SB in here

File: brands/peppa/src/less/hr.less [hunk 6]
```diff
@@ -0,0 +1,32 @@
+@import '../../../brand-globals/src/less/grid';
```
Comment (jodiedoubleday) [inline brands/peppa/src/less/hr.less]: grid?

File: brands/peppa/src/less/variables.less [hunk 7]
```diff
@@ -1 +1,313 @@
...
+@border-radius-large:       14px;
+// @border-radius-small:       2px;
+
+
+//== Progress Trackers
+//
+//##
+
+@progress-tracker-spacing:            3px;
+// @progress-tracker-bg:                 @brand-tertiary;
```
Comment (jodiedoubleday) [inline brands/peppa/src/less/variables.less]: consider removing commented out CSS

File: brands/paultons/src/less/footer.less (hunk 1 above)
Comment (nickine) [inline brands/paultons/src/less/footer.less]: SB Browser support minimum is IE11 - although for the sake of sanity I'll look at changing it

File: brands/peppa/src/less/colours.less (hunk 4 above)
Comment (nickine) [inline brands/peppa/src/less/colours.less]: Not crossed this yet - this is likely to be covered off when the navbar is done

File: brands/peppa/src/less/hr.less (hunk 6 above)
Comment (nickine) [inline brands/peppa/src/less/hr.less]: ha - pass

File: brands/peppa/src/less/variables.less (hunk 7 above)
Comment (nickine) [inline brands/peppa/src/less/variables.less]: I left a lot of the commented out stuff to allow the design team to see how the variables can be changed.

**Pull request merged**